src/pages/index.js

```diff
@@ -5,41 +5,21 @@ import PopupWithImage from "../components/PopupWithImage.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
 import "../pages/index.css";
-
-const initialCards = [
-  {
-    name: "Yosemite Valley",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-  },
-  {
-    name: "Lake Louise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Bald Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-  },
-  {
-    name: "Lago di Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-  },
-];
+import { Api } from "../components/API.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 
 //VARIABLES
 const cardWrapper = document.querySelector(".card-wrapper");
 const addCardButton = document.querySelector(".profile__add-button");
 const profileEditButton = document.querySelector(".profile__arrow");
+const profileAvatarButton = document.querySelector("#profile__imageButton");
+const profileModalButton = document.querySelector("#profileModal__button");
+const cardModalButton = document.querySelector("#cardModal__button");
+const avatarModalButton = document.querySelector("#avatarModal__button");
 
 const editForm = document.forms["profileModalForm"];
 const cardForm = document.forms["cardModalForm"];
+const avatarForm = document.forms["avatarForm"];
 
 //FORM VALIDATOR OBJECT
 const config = {
```

Comment:
From the 8th sprint all constants (`initialCards` and `config`) should be in file `constants.js` in folder `utils` to clean up the main file.